string  driveMode
int16   numberItems
int16[] motorIDs
int64[] command
